### DISCUSSION TITLE
Calculate the proportion of cell types in each group

### DISCUSSION THREAD

**Discussion by arturomarin** [satijalab/seurat#4151] [Q&A] [answered]
Hi, 

I am trying to reproduce the research of the article [Lee, Jeong Seok, et al (2020)](https://immunology.sciencemag.org/content/5/49/eabd1554). In this article they have 4 different groups of patients (with severe COVID, with mild COVID, with flu, and healthy). Is there some way to generate with Seurat a figure with the proportion of the cell types in this groups? in the article this is show in the figure 1D.

Thanks,

**Comment by timoast** [comment, marked as answer]
See this answer: https://bioinformatics.stackexchange.com/questions/6960/percentage-distribution-of-cells-in-all-clusters-based-on-their-treatment-condit/6962#6962
